src/main/java/gregtech/common/blocks/BlockMetalSheet.java

```diff
@@ -0,0 +1,61 @@
+package gregtech.common.blocks;
+
+import gregtech.api.block.VariantBlock;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.EntityLiving;
+import net.minecraft.util.IStringSerializable;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+
+public class BlockMetalSheet extends VariantBlock<BlockMetalSheet.SheetType> {
+
+    public BlockMetalSheet() {
+        super(net.minecraft.block.material.Material.IRON);
+        setTranslationKey("metal_sheet");
+        setHardness(2.0f);
+        setResistance(5.0f);
+        setSoundType(SoundType.METAL);
+        setDefaultState(getState(SheetType.WHITE));
+    }
+
+    @Override
+    public boolean canCreatureSpawn(IBlockState state, IBlockAccess world, BlockPos pos, EntityLiving.SpawnPlacementType type) {
+        return false;
+    }
+
+    public enum SheetType implements IStringSerializable {
+        WHITE("white", 1),
+        ORANGE("orange", 1),
+        MAGENTA("magenta", 1),
+        LIGHT_BLUE("light_blue", 1),
+        YELLOW("yellow", 1),
+        LIME("lime", 1),
+        PINK("pink", 1),
+        GRAY("gray", 1),
+        LIGHT_GRAY("light_gray", 1),
+        CYAN("cyan", 1),
+        PURPLE("purple", 1),
+        BLUE("blue", 1),
+        BROWN("brown", 1),
+        GREEN("green", 1),
+        RED("red", 1),
+        BLACK("black", 1);
```

Comment:
I think this enum can be reused for the other block classes. Maybe even just use the mc color enum. Also i don't see where the harvest level is used. And since all are the same it shouldnt me included in the constructor.
Maybe make 1 `ColoredBlock` class and just instantiate it multiple times with different registry names.

Reply:
Probably could reuse vanilla MC's `BlockColored`.

Reply:
Forgot to mention, but now they can be sprayed too.